packages/components/src/local/form-elements/checkboxes/types/props.d.ts

```diff
@@ -15,26 +11,19 @@ export default interface PropTypes {
   /**
    * The list of options available to the form section
    */
-  options: Array<string | number | FormOption>
+  options: (string | number)[]
   /**
    * The value to set the 'checked' parameter to
    */
   value: string | number | Array<string | number>
```

Comment:
The `value` parameter seems to loosely defined for me - it could cause complexity in rendering the component.  Can we force it to be an array, even if it's just one item?